src/test/kotlin/com/hedvig/underwriter/testhelp/databuilder/a.kt

```diff
@@ -68,7 +69,8 @@ class a {
             underwritingGuidelinesBypassedBy,
             memberId,
             originatingProductId,
-            signedProductId
+            agreementId,
+            contractId = contractId
```

Comment:
```suggestion
            contractId
```
it's okay but stranged to have just the last on as a named argument :)